cmd/cone/get_drop_task.go

```diff
@@ -410,7 +410,7 @@ func getEntitlementDetails(ctx context.Context, c client.C1Client, v *viper.Vipe
 				client.StringFromPtr(e.Entitlement.ID),
 			)
 			entitlementOptions = append(entitlementOptions, entitlementOptionName)
-			optionToEntitlementMap[entitlementOptionName] = &e.Entitlement
+			optionToEntitlementMap[entitlementOptionName] = &(e.Entitlement)
```

Comment:
What happened here?

Reply:
The linter didn't like `&e.Entitlement` because it thinks I'm making a reference to e (which gets reused every loop), not the entitlement on e.